.github/ISSUE_TEMPLATE.md

```diff
@@ -1,31 +1,28 @@
-## Summary
 
-*Briefly describe the issue and please DELETE the non applicable headings below. *
+### Observed behavior
 
-## System information
+description of the behavior that was observed, including screenshots or other references when applicable
 
-*Please specify the Kolibri version you were using and your operating system!*
+### Expected behavior
 
- - Version: ?
- - Operating system: ?
- - Browser: ?
+description of what behavior was expected but did not occur
 
-## Traceback or relevant snippet from server.log or browser console
+### User-facing consequences
 
-```
-INSERT TRACEBACK, LOG MESSAGES ETC. HERE
-```
+implications for a user of the application
 
-## How to reproduce
+### Errors and logs
 
-1. Steps
-1. To
-1. Reproduce
+```
+logs and errors here
+```
 
-## Screenshots
+### Steps to reproduce
 
-*If applicable*
+precise steps that someone else can follow in order to see this behavior
 
-## Real-life consequences (anything community should be aware of, for instance how it affects your deployment)
```

Comment:
I don't like that we remove the essential context - the real life consequences are things like "we have a school of 1000 students who cannot log in"

Reply:
We need information like this to better understand how to prioritize issue reports -- we introduced it in KA Lite because some issues that appear odd or small (like some browser quirk) would actually affect a lot of users.

Reply:
I changed this to "implications for a user of the application"

Reply:
clarified above, incorporating your question "who is a user?"